lib/events/conversion.js

```diff
@@ -2,73 +2,75 @@
 //
 // Badge events.
 
-import stringify  from 'json-stable-stringify';
-import validate   from '../validate';
+import stringify from 'json-stable-stringify'
+import validate from '../validate'
 
-const EVENT_TYPES = ['purchase', 'checkout', 'propensity_to_buy'];
+const EVENT_TYPES = ['purchase', 'checkout', 'propensity_to_buy']
 
-function validationsForEventType(eventType) {
+function validationsForEventType(/* eventType */) {
```

Comment:
Why this param is commented out? How the function works then?

Reply:
This argument is not used in the function, as I understand it - it was left for future developers who will update this function.
But now, after update linter's rules it shows an error - rule: no-unused-vars, and I commented out this